src/profile_manager/models.py

```diff
@@ -423,6 +428,32 @@ def post_delete_user(sender, instance, *args, **kwargs):
         instance.user.delete()
 
 
+@receiver(email_confirmed, sender=EmailConfirmationHMAC)
+def email_confirmed_handler(email_address, **kwargs):
+    """
+    After a user has confirmed their email, this will handle converting that email to the primary email
+    and therefore making it the User.email.
+
+    Then, we delete the non-primary email addresses of that user
```

Comment:
What is the non-primary email address?

Reply:
For context: https://github.com/bytedeck/bytedeck/pull/1300/files#r1161206616

When a user adds or updates their email address in the profile page, it creates new record for the allauth model `allauth.account.models.EmailAddress`.

So, after they confirmed their email, the new one becomes the primary and the old one becomes the non-primary, so we just delete that

Reply:
Can you add all that into the docstring?